docs/api-docs/partner/subscriptions/README.mdx

```diff
@@ -0,0 +1,67 @@
+# BigCommerce Subscription Foundation
+
+🚀 Fast track builds of custom subscription experiences on the BigCommerce platform. Save 100s of hours.
+
+💰 **Pre-integrated into Stripe Billing**, including authentication, merchant onboarding, subscription product management, subscription creation, and customer-facing subscription management.
+
+💅 Utilizes the Channels Toolkit to provide a natively integrated subscription channel that fits nicely within the BigCommerce control panel.
+
+| Product Management                       | Storefront Widget                     |
+| ---------------------------------------- | ------------------------------------- |
+| ![App Preview](sample-control-panel.png) | ![App Preview](sample-storefront.png) |
+
+## Get started
+
+Head to the [Subscription Foundation](https://developer.bigcommerce.com/docs/ae2455ab4d3d8-subscription-foundation) article to begin!
+
+## Deploy with Vercel
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fbigcommerce%2Fsubscription-foundation&env=NEXT_PUBLIC_APP_URL,DATABASE_URL,NEXT_PUBLIC_BC_APP_ID,BC_APP_CLIENT_ID,BC_APP_SECRET,NEXT_PUBLIC_STRIPE_CLIENT_ID,STRIPE_SECRET_KEY&envDescription=View%20the%20Subscription%20Foudation%20docs%20for%20information%20on%20each%20ENV%20variable.&envLink=https%3A%2F%2Fdeveloper.bigcommerce.com%2Fdocs%2Fae2455ab4d3d8-subscription-foundation&project-name=bigcommerce-subscription-foundation&demo-title=Subscription%20Foundation&demo-description=Helps%20create%20custom%20subscription%20experiences%20for%20BigCommerce.&demo-url=https%3A%2F%2Fgithub.com%2Fbigcommerce%2Fsubscription-foundation%2Fraw%2Fmain%2Fsample-storefront.png&demo-image=https%3A%2F%2Fgithub.com%2Fbigcommerce%2Fsubscription-foundation%2Fraw%2Fmain%2Fsample-storefront.png)
+
+## Key areas of codebase
+
+- `/backend`
+  - Where the auth and various API services live. These are uses within the internal API endpoints to keep routes secure and reach out to external APIs.
+- `/prisma`
+  - Where the DB models and initial seed data lives. Prisma uses this to generate DB clients for the app.
+- `/shared`
+  - Where the types and helper utilities are located.
+- `/src/pages`
+  - Where the sections of the app, including the UI for the BigCommerce app that will show inside the control panel, are managed.
+  - Next.js docs: https://nextjs.org/docs/basic-features/pages
+- `/src/pages/api`
+  - Where the APIs for the app are managed.
+  - Next.js docs: https://nextjs.org/docs/api-routes/introduction
+    - `/cart`, `/channel`, and `/product`: Contains endpoints that proxy to BigCommerce APIs.`
```

Comment:
```suggestion
    - `/cart`, `/channel`, and `/product`: Contains endpoints that proxy to BigCommerce APIs.
```

Reply:
Wasn't sure if the back ticks were necessary at the end of this line and the next two.

Reply:
@bc-tgomez thank you for finding these. I decided to close this ticket because we probably don't want to add this Readme article to the main branch. We can change the link as the requester indicated. I will add your changes to his PR.

Reply:
@bc-tgomez Looks like that PR has already been approved. I will put these changes in a different PR.